markdown/dev/howtos/ways-to-contribute/technical-writing/patterns/en.md

```diff
@@ -2,6 +2,13 @@
 title: Writing for freesewing.org
 ---
 
-You could write documentation for freesewing.org, our makers website.
+A good way of contributing to FreeSewing is writting documentation! 
 
-You would need good writing skills and a familiarity with sewing.
+We don't have specific guidelines on how to write documentation. You can write as you feel fit as long as it's relevant and follows [our Code of Conduct](/guides/code-of-conduct/) and [our community standards](https://freesewing.org/docs/various/community-standards/).
+
+Once in a while you might see some documentation using personal tone, such as `Whilst a covered button is recommended you can use whatever button you desire, feel free to experiment!`. While the impersonal tone is more common for documentation, the personal tone shows that FreeSewing is not a souless entity, but instead we are a community of awesome people! 
```

Comment:
With two exclamation marks in a single paragraph, plus the use of the word `whilst` as bonus, I think this is perhaps the best possible example of why *design by committee* leads to bad results.

I appreciate that @luizfzs is trying to keep everybody happy, but I really feel we're missing the forest for the trees here.